task/ecosystem-cert-preflight-checks/0.2/ecosystem-cert-preflight-checks.yaml

```diff
@@ -89,6 +90,25 @@ spec:
         printf "%s" "${_ARTIFACT_TYPE}" > "$(step.results.artifact-type.path)"
         echo "Introspection concludes that this artifact is of type \"${_ARTIFACT_TYPE}\"."
 
+    - name: generate-container-auth
```

Comment:
@bcrochet do you remember if Skopeo will properly select auth in the way `select-oci-auth` would?

If not, then I'd think the **generate-container-auth** step needs to happen first in this Task, given skopeo is used to read labels on the image in the **introspect** step

Reply:
Otherwise, the **generate-container-auth** can probably happen after **set-skip-for-bundles**

Reply:
Since skopeo uses containers/image and containers/storage, it does in fact select the auth in the way that is expected. Preflight does not process the auth in the same way, hence the need for this in the first place.